lib/node_modules/@stdlib/math/special/data/unary_function_database.json

```
@@ -0,0 +1,3386 @@
+      "uint32": "@stdlib/number/uint32/base/identity",
+      "float32": "@stdlib/math/base/special/absf",
+      "float64": "@stdlib/math/base/special/abs",
+      "complex64": "@stdlib/math/base/special/cabsf",
+      "complex128": "@stdlib/math/base/special/cabs",
+      "generic": "@stdlib/math/base/special/abs"
+    },
+    "policies": {
+      "output": "real_and_generic",
+      "casting": "none"
+    }
+  },
+  "abs2": {
+    "input_dtypes": "numeric_and_generic",
+    "output_dtypes": "real_and_generic",
+    "excluded_dtypes": [
+      "float16",
+      "int64",
+      "uint64",
+      "uint8c",
+      "complex32"
+    ],
+    "scalar_kernels": {
+      "int8": "@stdlib/math/base/special/abs2f",
+      "int16": "@stdlib/math/base/special/abs2f",
+      "int32": "@stdlib/math/base/special/abs2f",
+      "uint8": "@stdlib/math/base/special/abs2f",
+      "uint16": "@stdlib/math/base/special/abs2f",
+      "uint32": "@stdlib/math/base/special/abs2f",
+      "float32": "@stdlib/math/base/special/abs2f",
+      "float64": "@stdlib/math/base/special/abs2",
+      "complex64": "@stdlib/math/base/special/cabs2f",
+      "complex128": "@stdlib/math/base/special/cabs2",
+      "generic": "@stdlib/math/base/special/abs2"
+    },
+    "policies": {
+      "output": "real_and_generic",
+      "casting": "none"
+    }
+  },
+  "absgammalnf": {
+    "input_dtypes": "real_and_generic",
+    "output_dtypes": "real_floating_point_and_generic",
+    "excluded_dtypes": [
+      "float16",
+      "uint8c",
+      "int64",
+      "uint64"
+    ],
+    "scalar_kernels": {
+      "int8": "@stdlib/math/base/special/absgammalnf",
+      "int16": "@stdlib/math/base/special/absgammalnf",
+      "int32": "@stdlib/math/base/special/absgammalnf",
+      "uint8": "@stdlib/math/base/special/absgammalnf",
+      "uint16": "@stdlib/math/base/special/absgammalnf",
+      "uint32": "@stdlib/math/base/special/absgammalnf",
+      "float32": "@stdlib/math/base/special/absgammalnf",
+      "float64": "@stdlib/math/base/special/absgammalnf",
+      "generic": "@stdlib/math/base/special/absgammalnf"
+    },
+    "policies": {
+      "output": "real_floating_point",
+      "casting": "none"
+    }
+  },
+  "acos": {
+    "input_dtypes": "real_and_generic",
+    "output_dtypes": "real_floating_point_and_generic",
+    "excluded_dtypes": [
+      "float16",
+      "uint8c",
+      "int64",
+      "uint64"
+    ],
+    "scalar_kernels": {
+      "int8": "@stdlib/math/base/special/acosf",
+      "int16": "@stdlib/math/base/special/acosf",
+      "int32": "@stdlib/math/base/special/acosf",
+      "uint8": "@stdlib/math/base/special/acosf",
+      "uint16": "@stdlib/math/base/special/acosf",
+      "uint32": "@stdlib/math/base/special/acosf",
+      "float32": "@stdlib/math/base/special/acosf",
+      "float64": "@stdlib/math/base/special/acos",
+      "generic": "@stdlib/math/base/special/acos"
+    },
+    "policies": {
+      "output": "real_floating_point",
```

```suggestion
      "output": "real_floating_point_and_generic",
```

Anywhere where `generic` is an accepted input dtype, it should also be an accepted output dtype.